content/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lifecycle badges
+---
+
+![](/images/lifecycle/lifecycle.png)
+
+### Experimental ![](https://img.shields.io/badge/lifecycle-experimental-orange.svg) {#experimental}
+
+Very early days, a lot of churn in search of a good API.  Not on CRAN. Might not go anywhere. Use with care.
+
+### Maturing ![](https://img.shields.io/badge/lifecycle-maturing-blue.svg) {#maturing}
+
+API roughed out, but finer details likely to change. Will strive to maintain backward compatibility, but need wider usage in order to get more feedback.
+
+### Stable ![](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg) {#stable}
+
+We're happy with the API, and the unlikely to be major changes. Backward incompatible changes will only be made if absolutely critical, and will be accompanied by a change in the major version.
+
+### Retired ![](https://img.shields.io/badge/lifecycle-retired-orange.svg) {#retired}
+
+Known better replacement available elsewhere. Will remain  available on CRAN.
```

Comment:
A retired package is no longer under active development, and a known better alternative is available. We will only make the necessary changes to ensure that retired packages remain on CRAN. No new features will be added, and only the most critical of bugs will be fixed.